packages/core/src/components/menu-about/menu-about.tsx

```diff
@@ -34,8 +34,7 @@ export class MenuAbout {
   /**
    * Active tab
    */
-  @Prop({ mutable: true, reflect: true }) activeTabLabel: string =
-    this.i18nImprintLabel;
+   @Prop({ mutable: true }) activeTabLabel: string;
```

Comment:
If we want to remove this prop we should mark the i18nImprintLabel as deprecated or remove it complete because i think this is an leftover from the refactoring